### PR TITLE
[NUI][NUI.Devel.Tests] Fix build errors of NUI.Devel following latest codes.

### PR DESCRIPTION
**Pull request by AchoWang**
### API Changes ###

 - ACR:

### CODE REVIEW

**Review by zhouleonlei — APPROVED** [review]
lgtm

**Review by xiaowei-guan — APPROVED** [review]
LGTM

**Review by huayongxu — APPROVED** [review]
lgtm

**Review by elishateng — APPROVED** [review]
LGTM

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by zg2nets — APPROVED** [review]
looks good

File: test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs [hunk 1]
```diff
@@ -228,72 +93,6 @@ public void ViewAccessibilityAccessibilityRangeContent()
...
-        }
-
-        [Test]
-        [Category("P1")]
-        [Description("ViewAccessibility.View.AccessibilityAttributes.Clear.")]
-        [Property("SPEC", "Tizen.NUI.ViewAccessibility.View.AccessibilityAttributes.Clear MR")]
-        [Property("SPEC_URL", "-")]
-        [Property("CRITERIA", "MR")]
-        [Property("AUTHOR", "guowei.wang@samsung.com")]
-        public void ViewAccessibilityViewClearAccessibilityAttributes()
```
Comment (aswigon) [inline test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs:271]: This test has just been updated: https://github.com/Samsung/TizenFX/pull/4404

**Review by huayongxu — APPROVED** [review]
lgtm

File: test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/Accessibility/TSAccessibility.cs [hunk 2]
```diff
@@ -26,31 +26,6 @@ public void Destroy()
...
-        [Property("SPEC_URL", "-")]
-        [Property("CRITERIA", "MR")]
-        [Property("AUTHOR", "guowei.wang@samsung.com")]
-        public void AccessibilityGetStatus()
-        {
-            tlog.Debug(tag, $"AccessibilityGetStatus START");
-
-            try
-            {
-                var result = Accessibility.Accessibility.GetStatus();
```
[the remark this answers is not in the document]
Comment (AchoWang) [reply test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/Accessibility/TSAccessibility.cs:42]: Done

> Can be changed to `var result = Accessibility.Accessibility.IsEnabled;` (also `.IsScreenReaderEnabled`).

Done

File: test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs (hunk 1 above)
Comment (AchoWang) [reply test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs]: > This test has just been updated: #4404

AccessibilityAttributes is protected,  we cannot call it directly, so I rewrite the relevant case, please check.

File: test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs [hunk 3]
```diff
@@ -228,72 +93,6 @@ public void ViewAccessibilityAccessibilityRangeContent()
...
         }
 
-        [Test]
-        [Category("P1")]
-        [Description("ViewAccessibility.View.AccessibilityAttributes.Remove.")]
-        [Property("SPEC", "Tizen.NUI.ViewAccessibility.View.AccessibilityAttributes.Remove M")]
-        [Property("SPEC_URL", "-")]
-        [Property("CRITERIA", "MR")]
-        [Property("AUTHOR", "guowei.wang@samsung.com")]
-        public void ViewAccessibilityViewRemoveAccessibilityAttribute()
```
[the remark this answers is not in the document]
Comment (AchoWang) [reply test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs]: > This test has just been updated: #4404

AccessibilityAttributes is protected,  we cannot call it directly, so I rewrite the relevant case, please check.

[the remark this answers is not in the document]
Comment (aswigon) [reply test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs]: `AccessibilityAttributes` could theoretically be made `public`. `protected` was chosen based on the fact that all usages of attributes in NUI were from within the `View` class. But this way is very elegant as well.

File: test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs (hunk 1 above)
Comment (aswigon) [reply test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs]: OK

**Review by dongsug-song — APPROVED** [review]
approved

**Pull request merged**